omod/src/main/webapp/resources/jsonforms/HMIS-ACP-003-HIVCAREARTCARD-SUMMARYPAGE.json

```diff
@@ -0,0 +1,787 @@
+                {
+                  "type": "date",
+                  "allowFutureDates": "false"
+                }
+              ],
+              "hide": {
+                "hideWhenExpression": "isEmpty(pmtctChecked) || !arrayContains(['dcd695dc-30ab-102d-86b0-7a5022ba4115'], perpChecked)"
+              }
+            },
+            {
+              "label": "Duration in Months",
+              "type": "obs",
+              "id": "durationInMonths",
+              "questionOptions": {
+                "rendering": "number",
+                "concept": "d11d4ad1-4aa2-4f90-8f2c-83f52155f0fc",
+                "conceptMappings": [],
+                "answers": []
+              },
+              "validators": [],
+              "hide": {
+                "hideWhenExpression": "isEmpty(pmtctChecked) || !arrayContains(['dcd695dc-30ab-102d-86b0-7a5022ba4115'], prepChecked)"
+              }
+            },
+            {
+              "label": "ART(not in transfer)",
+              "type": "obs",
+              "id": "artChecked",
+              "questionOptions": {
+                "concept": "a75ab6b0-dbe7-4037-93aa-f1dfd3976f10",
+                "rendering": "checkbox",
+                "answers": [
+                  {
+                    "concept": "dcd695dc-30ab-102d-86b0-7a5022ba4115",
+                    "label": "ART"
+                  }
+                ]
+              },
+              "hide": {
+                "hideWhenExpression": "isEmpty(encART) || !arrayContains(['dcd695dc-30ab-102d-86b0-7a5022ba4115'], perpChecked)"
+              }
+            },
+            {
+              "label": "ART Regimen Start Date",
+              "type": "obs",
+              "id": "regimenStartDatePreP",
+              "questionOptions": {
+                "concept": "999dea3b-ad8b-45b4-b858-d7ab98de486c",
+                "rendering": "date"
+              },
+              "validators": [
+                {
+                  "type": "date",
+                  "allowFutureDates": "false"
+                }
+              ],
+              "hide": {
+                "hideWhenExpression": "isEmpty(artChecked) || !arrayContains(['dcd695dc-30ab-102d-86b0-7a5022ba4115'], perpChecked)"
+              }
+            },
+            {
+              "label": "Duration in Months",
+              "type": "obs",
+              "id": "durationInMonths",
+              "questionOptions": {
+                "rendering": "number",
+                "concept": "d11d4ad1-4aa2-4f90-8f2c-83f52155f0fc",
+                "conceptMappings": [],
+                "answers": []
+              },
+              "validators": [],
+              "hide": {
+                "hideWhenExpression": "isEmpty(artChecked) || !arrayContains(['dcd695dc-30ab-102d-86b0-7a5022ba4115'], prepChecked)"
+              }
+            },
+            {
+              "label": "Approach",
+              "type": "programState",
+              "id": "dsdmApproach",
```

Comment:
@jabahum do we still DSDM approach here??

Reply:
i am going to remove it since its also not on the summary page. sorry i missed it. Thnks

Reply:
@jabahum have also added some other specify fields you missed out.